src/components/Settings/ChangePasswordForm.js

```diff
@@ -47,17 +47,16 @@ const ChangePasswordForm = () => {
         label={intl.messages['common.currentPassword']}
         name="currentPassword"
         type="password"
-        data-testid="current-password-input-settings"
       />
       <Form.Input
         label={intl.messages['common.newPassword']}
         name="password"
         type="password"
-        data-testid="password-input-settings"
       />
       <Form.Button
         isLoading={isLoading}
         data-testid="submit-password-button"
+        role="submit-password"
```

Comment:
what if you just use `getByRole('button', ...)`?

Reply:
https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-querying-by-text